package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "luxon",
-  "version": "1.22.2",
+  "version": "1.23.0",
```

Comment:
Please remove this, I do that separately at release time.

Reply:
I've removed the version bump.